apps/backend/src/modules/projects/project.service.ts

```diff
@@ -68,7 +68,6 @@ class ProjectService implements Service {
 			items: projects.items.map((item) => item.toObject()),
 		};
 	}
-
 	public async findAllbyName(
```

Comment:
Better add that line back here, so this file won't be changed :sweat_smile:

Reply:
The line has now been added back in :D now the PR is cleaner, thanks!